Objects/unicodeobject.c

```diff
@@ -13078,16 +13078,24 @@ unicode_startswith_impl(PyObject *self, PyObject *subobj, Py_ssize_t start,
 
 
 /*[clinic input]
-@text_signature "($self, prefix[, start[, end]], /)"
-str.endswith as unicode_endswith = str.startswith
+@text_signature "($self, suffix[, start[, end]], /)"
+str.endswith as unicode_endswith
+
+    suffix as subobj: object
+        A string or a tuple of strings to try.
+    start: slice_index(accept={int, NoneType}, c_default='0') = None
+        Optional start position. Default: start of the string.
```

Comment:
for a small simplification?

```suggestion
        Optional start position. Default: 0.
```

Reply:
That was my initial suggestion, but I like the symmetry you get by spelling out both param docstrings.

- Default: 0
- Default: end of the string

vs.

- Default: start of the string
- Default: end of the string

Reply:
I personally lean towards simplicity over symmetry here, but I don't have a strong opinion :)

Reply:
Ok, I'll say thanks, but no thanks 😄